spv/errors.go

```diff
@@ -46,4 +46,25 @@ var (
 
 	// ErrMissingOutput is returned when checking fees if an output in a parent tx is missing.
 	ErrMissingOutput = errors.New("expected output used in payment tx missing")
+
+	// ErrProofOrInputMissing returns if a path from tip to beginning/anchor is broken.
+	ErrProofOrInputMissing = errors.New("break in the ancestry missing either a parent transaction or a proof")
+
+	// ErrTriedToParseZeroBytes returns weh we attempt to parse a slice of bytes of zero length.
```

Comment:
typo, does the linter not catch them?

Reply:
also what do you mean by `callback bytes`?

Reply:
mapi response it should read - updated.